src/formatted-codegen.js

```diff
@@ -732,6 +744,13 @@ export class ExtensibleCodeGen {
       });
   }
 
+  reduceForAwaitStatement(node, { left, right, body }) {
+    left = node.left.type === 'VariableDeclaration' ? noIn(markContainsIn(left)) : left;
+    return objectAssign(
+      seq(this.t('for'), this.sep(Sep.AFTER_FOROF_FOR), this.sep(Sep.AFTER_FORAWAIT_AWAIT), this.sep(Sep.AWAIT), this.paren(seq(left.startsWithLet ? this.paren(left, Sep.FOR_OF_LET_PAREN_BEFORE, Sep.FOR_OF_LET_PAREN_AFTER) : left, this.sep(Sep.BEFORE_FOROF_OF), this.t('of'), this.sep(Sep.AFTER_FOROF_FOR), this.p(node.right, Precedence.Assignment, right)), Sep.FOR_OF_PAREN_BEFORE, Sep.FOR_OF_PAREN_AFTER), this.sep(Sep.BEFORE_FOROF_BODY), body, this.sep(Sep.AFTER_STATEMENT(node))),
```

Comment:
`this.sep(Sep.AFTER_FORAWAIT_AWAIT), this.sep(Sep.AWAIT)` are in the wrong order.